ngi_pipeline/engines/piper_ngi/local_process_tracking.py

```diff
@@ -81,7 +81,12 @@ def update_charon_with_local_jobs_status(quiet=False, config=None, config_file_p
             try:
                 if piper_exit_code == 0:
                     # 0 -> Job finished successfully
-                    set_status = "ANALYZED" # For the sample level
+                    if workflow == "merge_process_variantcall":
+                        status_field = "analysis_status"
+                        set_status = "ANALYZED" # sample level
```

Comment:
These differences between the various sample/seqrun and genotype_status/analysis_status field values are annoying and probably not necessary. We could consider making them more uniform? At least for values that mean the same thing -- `ANALYZED` means the same as `DONE`, for instance.